flake8_django/checkers/model_content_order.py

```diff
@@ -0,0 +1,107 @@
+from ast import Assign, Call, ClassDef, FunctionDef
+
+from .base_model_checker import BaseModelChecker
+from .issue import Issue
+
+
+class DJ09(Issue):
+    code = 'DJ09'
+    description = 'Order of model inner classes and standard methods does not follow Django style guide: {elem_type} should come before {before}'
+
+    def __init__(self, elem, elem_type, before):
+        super().__init__(elem.lineno, elem.col_offset, None)
+        self.description = self.description.format(
+            elem_type=elem_type,
+            before=before,
+        )
+
+
+def is_assignment_call(node):
+    # check if assigning the return value of a function call to a target which is not in all uppercase letters
+    # (because that would be a constant)
+    return isinstance(node, Assign) and isinstance(node.value, Call) and not node.targets[0].id.isupper()
+
+
+def is_manager_declaration(node):
+    return isinstance(node, Assign) and node.targets[0].id == 'objects'
+
+
+def is_meta_declaration(node):
+    return isinstance(node, ClassDef) and node.name == 'Meta'
+
+
+def is_save_declaration(node):
+    return isinstance(node, FunctionDef) and node.name == 'save'
+
+
+def is_str_declaration(node):
+    return isinstance(node, FunctionDef) and node.name == '__str__'
+
+
+def is_url_declaration(node):
```

Comment:
We can remove this check, since we are using [partials](https://docs.python.org/3.8/library/functools.html#functools.partial), and updating `is_method` to receive the name parameter.